package.json

```diff
@@ -1,18 +1,19 @@
 {
   "name": "bloom-forms",
-  "version": "1.3.2",
+  "version": "1.3.3",
   "description": "form functionality used in bloom packages",
   "main": "index.js",
   "scripts": {
     "build": "./node_modules/.bin/webpack --config=webpack.config.js",
     "lint": "./node_modules/.bin/eslint --config ./.eslintrc --fix ./src/* --ext .js,.jsx",
     "prettier": "./node_modules/.bin/prettier --config ./.prettierrc --write './src/**/*.{js,jsx,scss}'",
     "test": "npm run test-redux && npm run test-components && npm run test-validator",
-    "test-components": "npm run test-form && npm run test-inputs",
-    "test-form": "./node_modules/.bin/mocha  --compilers js:babel-core/register tests/helpers/virtual-dom.js tests/formTest.js --require ignore-styles",
-    "test-inputs": "./node_modules/.bin/mocha --compilers js:babel-core/register tests/helpers/virtual-dom.js tests/inputsTest.js  --require ignore-styles",
-    "test-redux": "./node_modules/.bin/mocha tests/reduxTest --compilers js:babel-core/register",
-    "test-validator": "./node_modules/.bin/mocha tests/validatorTest --compilers js:babel-core/register"
+    "test-components": "npm run test-form && npm run test-inputs && npm run test-select",
```

Comment:
You should be able to call binaries directly from the `./node_modules/.bin/` directly in the scripts section.  IE:
```
"test-form": "mocha  --compilers js:babel-core/register tests/helpers/virtual-dom.js tests/form-test.js --require ignore-styles",
```

Reply:
@seansean11 you're right -- I found that it was defaulting to my global version of mocha instead of the local one installed, which is why I've defaulted to this syntax.

Is there a better way to indicate local over global preference?